### PR TITLE
Slightly better code layout

### PR DESCRIPTION
**Pull request by murbanski**

### CODE REVIEW

File: db/fixtures/development/07_milestones.rb [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
   Project.all.each do |project|
-    (1..5).each  do |i|
+    (1..5).each do |i|
       milestone_params = {
         title: "v#{i}.0",
         description: FFaker::Lorem.sentence,
         state: ['opened', 'closed'].sample,
       }
 
-      milestone = Milestones::CreateService.new(
```
Comment (murbanski) [inline db/fixtures/development/07_milestones.rb]: Unused variable?

**Comment by stanhu** [comment]
Please rebase master again, and the tests should pass.

**Comment by murbanski** [comment]
@stanhu How about now?

**Comment by stanhu** [comment]
Build is still failing. I think you may still need to rebase master.

**Comment by jvanbaarsen** [comment]
This merge request has been closed because a request for more information has not been reacted to for more than 2 weeks. If you respond and conform to the merge request guidelines in our [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#pull-requests) we will reopen this merge request.

**Pull request closed without merging**